doc/developing/download.rst

```diff
@@ -0,0 +1,25 @@
+..
+    :copyright: Copyright (c) 2014-2021 ftrack
+
+.. _developing/download:
+
+********
+Download
+********
+
+.. highlight:: bash
+
+The framework is 100% open source and accessible on GitHub.
+
+Git clone base repositories::
+
+    gh repo clone ftrackhq/ftrack-connect-pipeline
```

Comment:
@henriknorin-ftrack 
git clone is better than gh as it is gihub specific only and we want to be as generic as possible.